src/platform/nxp/k32w/k32w0/PlatformManagerImpl.cpp

```diff
@@ -59,6 +61,13 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
     // Initialize the configuration system.
     err = Internal::K32WConfig::Init();
     SuccessOrExit(err);
+
+    if (Chip_GetType() != CHIP_K32W061)
+    {
+        err = CHIP_ERROR_INTERNAL;
```

Comment:
A log may be nice to see on  the console (if logging works at all if run on a different device) - saying that invalid platform or something similar

Reply:
Agree, added a log error - we had a situation where someone tried to run the example using JN5189 instead of K32W061, in this case the logging still works.